specification/loadtestservice/resource-manager/Microsoft.LoadTestService/stable/2022-12-01/loadtestservice.json

```diff
@@ -2,985 +2,1075 @@
+            "name": "loadTestName",
+            "in": "path",
+            "description": "Load Test name",
+            "required": true,
+            "type": "string",
+            "x-ms-parameter-location": "method"
           }
         ],
         "responses": {
           "200": {
-            "description": "The operation was successful. The response contains a list of outbound network dependency endpoints.",
+            "description": "Azure operation completed successfully.",
             "schema": {
               "$ref": "#/definitions/OutboundEnvironmentEndpointCollection"
             }
           },
           "default": {
-            "description": "Error response describing why the operation failed.",
+            "description": "An unexpected error response.",
             "schema": {
-              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
             }
           }
         },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
+        "x-ms-examples": {
+          "Lists the endpoints that agents may call as part of load testing.": {
+            "$ref": "./examples/LoadTests_ListOutboundNetworkDependenciesEndpoints.json"
+          }
         }
       }
     }
   },
   "definitions": {
-    "LoadTestResourcePageList": {
-      "description": "List of resources page result.",
-      "type": "object",
-      "properties": {
-        "value": {
-          "description": "List of resources in current page.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/LoadTestResource"
-          }
-        },
-        "nextLink": {
-          "description": "Link to next page of resources.",
-          "type": "string"
-        }
-      }
-    },
-    "LoadTestResource": {
+    "CheckQuotaAvailabilityResponse": {
       "type": "object",
-      "description": "LoadTest details",
+      "description": "Check quota availability response object.",
       "properties": {
         "properties": {
-          "description": "Load Test resource properties",
-          "$ref": "#/definitions/LoadTestProperties",
+          "$ref": "#/definitions/CheckQuotaAvailabilityResponseProperties",
+          "description": "Check quota availability response properties.",
           "x-ms-client-flatten": true
-        },
-        "identity": {
-          "$ref": "../../../../../common-types/resource-management/v5/managedidentity.json#/definitions/ManagedServiceIdentity",
-          "description": "The type of identity used for the resource."
         }
       },
       "allOf": [
         {
-          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/Resource"
```

Comment:
I don't think this should be inheriting Resource, since its a POST response. Should be okay to make a breaking change with preview api versions right.

Reply:
I tend to agree with Tim here. The inheritence part feels wrong semantically. Maybe use spread `...` instead. It will spread all of the properties directly into this model.

```
model QuoteNameCheckRequest {
  ...TrackedResource<EmployeeProperties>;
}
```

Reply:
Used spread instead of inheritance as suggested.

Reply:
It is not typical though to return resource bodies in POST responses. One reason is that if that response really represents an Azure resource with its own GET or PUT operation, seeing resource bodies should need RBAC read permission, and should be done with a GET request, to get the right permission check.

Reply:
This is already published to GA. Is it possible to amend it now?